src/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List</title>
+</head>
+<body>
+  <div class="todo">
+    <div class="page-title">
+    </div>
+    <form class="input">
+    </form>
+    <ul class="todo-list">
+    </ul>
+    <div class="clear-link">
+      <a href="#" class="link">Clear all completed</a>
+    </div>
+  </div>
```

Comment:
Thank you for noticing it. I kept a div tag instead of a section tag because inside the section tag we have to put a title 'usually h2 tag' and a p tag for the description as it makes the page more accessible.